common/guestchain/src/manager.rs

```diff
@@ -507,4 +526,49 @@ fn test_generate() {
         false,
     )
     .unwrap();
+
+    let update_chain_config = UpdateConfig {
+        min_validators: NonZeroU16::new((mgr.validators().len() + 1) as u16),
+        max_validators: None,
+        min_validator_stake: None,
+        min_total_stake: None,
+        min_quorum_stake: None,
+        min_block_length: None,
+        max_block_age_ns: None,
+        min_epoch_length: None,
+    };
+    assert_eq!(
+        Err(UpdateConfigError::MinValidatorsHigherThanExisting),
+        mgr.update_config(update_chain_config)
+    );
+
+    let update_chain_config = UpdateConfig {
+        min_validators: None,
+        max_validators: NonZeroU16::new(u16::from(config.max_validators) - 1),
+        min_validator_stake: None,
+        min_total_stake: Some(NonZeroU128::new(total_stake + 2).unwrap()),
```

Comment:
Same here and below you could get rid of Some and unwrap.

Reply:
gotcha